torch/_inductor/fx_passes/split_cat.py

```diff
@@ -215,3 +594,16 @@ def _split_cat_init():
             pattern=pattern, extra_check=config_flag("split_cat_fx_passes")
         )
         pattern.register(normalize_split_pass)
+
+    for pattern in [
+        CallFunctionVarArgs(
+            torch.cat,
+        ),
+        CallFunctionVarArgs(
+            torch.stack,
+        ),
+    ]:
+        pattern = MergeSplitCat(
+            pattern=pattern, extra_check=config_flag("split_cat_fx_passes")
+        )
+        pattern.register(merge_split_cat_pass)
```

Comment:
That's a simple change that I can make. Although I wanted a class structure, since there are so many functions - class helps group all related functions together. And I'm not sure if the decorator would work on a class method.